src/mongocxx/include/mongocxx/v_noabi/mongocxx/client_encryption.hpp

```diff
@@ -134,7 +134,7 @@ class client_encryption {
     ///
     /// Encrypts a Match Expression or Aggregate Expression to query a range index.
     ///
-    /// @note Only supported when queryType is "rangePreview" and algorithm is "RangePreview".
+    /// @note Only supported when queryType is "range" and algorithm is "Range".
     ///
```

Comment:
Remove "The Range algorithm is experimental only." note in this file and others. Range is now GA for MongoDB 8.0.